src/main/java/net/sf/jabref/importer/fileformat/SixpackImporter.java

```diff
@@ -139,20 +141,18 @@ public boolean isRecognizedFormat(InputStream stream) throws IOException {
                 }
                 BibtexEntryType typ = EntryTypes.getType(fields[1].toLowerCase());
                 if (typ == null) {
-                    String type = "";
                     if (fields[1].equals("Masterthesis")) {
-                        type = "mastersthesis";
+                        typ = BibtexEntryTypes.MASTERSTHESIS;
                     }
                     if (fields[1].equals("PhD-Thesis")) {
-                        type = "phdthesis";
+                        typ = BibtexEntryTypes.PHDTHESIS;
                     }
                     if (fields[1].equals("miscellaneous")) {
-                        type = "misc";
+                        typ = BibtexEntryTypes.MISC;
                     }
                     if (fields[1].equals("Conference")) {
```

Comment:
Yeah, I asked myself that as well in the back of my head, totally focused on replacing stuff. :-)

But I'll change it now. Good point!